lib/serverless/aws-lambda.js

```diff
@@ -122,13 +126,26 @@ class AwsLambda {
       this.wrapFatal()
     }
 
-    return shim.bindCreateTransaction(wrappedHandler, new specs.TransactionSpec({ type: shim.BG }))
+    const nrHandler = shim.bindCreateTransaction(wrappedHandler, new specs.TransactionSpec({ type: shim.BG }))
+
+    // Response-streaming handlers are identified by symbol properties on the function.
+    // We propagate any symbols if they're present, so that the handler keeps its signature
+    // for any AWS features that rely on symbols
+    for (const symbol of Object.getOwnPropertySymbols(handler)) {
+      logger.trace(`Setting symbol ${symbol.toString()} on handler`)
+      nrHandler[symbol] = handler[symbol]
+    }
+
+    return nrHandler
 
     function wrappedHandler() {
       const args = shim.argsToArray.apply(shim, arguments)
+      logger.trace('handler args', args)
 
       const event = args[0]
-      const context = args[1]
+      logger.trace('Is this a response streaming Lambda function?', isStreamHandler)
+      const context = isStreamHandler === true ? args[2] : args[1]
+      logger.trace('Context', context)
```

Comment:
do we want this to ship?

Reply:
We found it useful in debugging, but I'm not 100% in favor of logging out this context. I could see the value in logging out the context type